contrib/windows/msys_build.sh

```diff
@@ -206,3 +206,4 @@ cat Make.user
 make -j3 VERBOSE=1
 make build-stats
 #make debug
+make prefix=/c/projects/julia/usr VERBOSE=1 -C examples
```

Comment:
All I want is for it to work. Here are a few things I've tried:

- `make prefix=/c/projects/julia/usr VERBOSE=1 -C examples` in msys_build.sh --- works but hard-codes a path, not ideal.
- `make VERBOSE=1 -C examples` in msys_build.sh --- fails attempting to find `/c/projects/julia/examples/julia-251f61c05f/bin/julia.exe`
- `C:\msys64\usr\bin\sh.exe --login -c 'make prefix=/c/projects/julia/usr VERBOSE=1 -C examples'` in appveyor.yml --- fails saying it can't find `examples`.

What I need to do is (1) trigger a `make` in examples, (2) that process needs to be able to run `julia julia-config.jl`. Anything that accomplishes that is fine with me.